packages/medusa/src/api/routes/admin/promotions/validators.ts

```diff
@@ -0,0 +1,13 @@
+import { IsOptional, IsString } from "class-validator"
+import { FindParams, extendedFindParamsMixin } from "../../../../types/common"
+
+export class AdminGetPromotionsPromotionParams extends FindParams {}
```

Comment:
we might finally start getting rid of the cyclic deps import and move toward newer version of ES :D